packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -330,6 +331,9 @@ contract AssetHolder is IAssetHolder {
 
         // first increase payouts according to guarantee
         for (uint256 i = 0; i < guarantee.destinations.length; i++) {
+            if (balance == 0) {
+                    break;
```

Comment:
Indenting seems too much?

Reply:
Good spot. I'm sad that prettier didn't spot or fix this for me.
I'm going to tackle in a separate PR.

Reply:
go on then eb0179a